Added link to alexjustesen/speedtest-tracker (#57)
benphelps/homepage-docs: src/pages/en/services/speedtest-tracker.md

No extra configuration is required. 

> Note that the only version of this software that currently supports an API are [alexjustesen/speedtest-tracker](https://github.com/alexjustesen/speedtest-tracker) and [henrywhitaker3/Speedtest-Tracker](https://github.com/henrywhitaker3/Speedtest-Tracker).

Reviewer: ```suggestion
This widget is compatible with both [alexjustesen/speedtest-tracker](https://github.com/alexjustesen/speedtest-tracker) and [henrywhitaker3/Speedtest-Tracker](https://github.com/henrywhitaker3/Speedtest-Tracker).
```
